### PR TITLE
Updates for rse.diagnijmegen.nl

### LINKED ISSUE #167: Algorithm update for RSE webpage on grand-challenge
The RSE webpage on grand-challenge needs an update.

### PR DESCRIPTION
**Pull request by silvandeleemput**
Closes #167

### CODE REVIEW

**Comment by martvanrijthoven** [comment]
Can this be closed?

**Comment by silvandeleemput** [comment]
@MikeOverkamp-diag I see that you did some updates as well, do you think you could resolve the conflict? Maybe @jmsmkn can subsequently approve the changes?

File: content/pages/software/grand-challenge.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ finished: false
 picture: software/grand-challenge.png
 template: project-single
 groups: rse, diag
-people: James Meakin, Sjoerd Kerkstra, Paul Konstantin Gerke, Harm van Zeeland, Karel Gerbrands, Bram van Ginneken
+people: James Meakin, Sjoerd Kerkstra, Paul Konstantin Gerke, Harm van Zeeland, Bram van Ginneken, Sil van de Leemput, Mike Overkamp
```
Comment (jmsmkn) [inline content/pages/software/grand-challenge.md:7]: Could you add Miriam, Anne and Chris?

Comment (silvandeleemput) [reply content/pages/software/grand-challenge.md:7]: I'll do that

**Pull request merged**